site/docs/voice/bxml/startTranscription.mdx

```diff
@@ -0,0 +1,208 @@
+| metadata     | Details about the real-time transcription this message is for. See further details below. |
+| customParams | (optional) (`start` message only) If any `<CustomParam/>` elements were specified in the `<StartTranscription>` request, they will be copied here as a map of `name : value` pairs |
+
+#### Metadata Parameters
+
+| Parameter                     | Description |
+|:------------------------------|:------------|
+| accountId                     | The user account associated with the call |
+| callId                        | The call id associated with the real-time transcription |
+| realtimetranscriptionId       | The unique id of the real-time transcription |
+| transcriptionName             | The user supplied name of the real-time transcription |
+| tracks                        | A list of one or more tracks being transcribed in real-time |
+| tracks.name                   | The name of the track being transcribed, will be used to identify which transcription updates belong to which track |
+| stabilized                    | Whether transcription updates will be sent only after they have become stable or not |
+
+### Transcription Message Parameters
+
+| Parameter | Description            |
+|:----------|:-----------------------|
+| eventType | Will always be `transcription` |
+| track     | The name of the track this transcription update is for, will be one of the names specified in the `start` message |
+| startTime   | The time at which this segment started |
+| endTime   | The time at which this segment ended |
+| isPartial   | Indicates if the segment is complete |
+| language   | The detected language of the segment |
+| transcript   | The transcription of this segment as a flattened string |
+| items   | The list of items making up this segment |
+| items.content   | A word or punctuation |
+| items.startTime   | The time at which this item started |
+| items.endTime   | The time at which this item ended |
+| items.confidence   | The confidence score associated with a word or phrase in your transcript. |
+| items.stable   | Indicates whether the specified item is stable (true) or if it may change when the segment is complete (false). |
+| items.type   | Either `PRONUNCIATION` or `PUNCTUATION` |
+
+## Examples
+
+### A `start` Websocket Message
+
+```json
+{
+  "eventType": "start",
+  "metadata": {
+    "accountId": "5555555",
+    "callId": "c-2a913f94-7fa91773-a426-4118-8b8b-b691ab0a0ae1",
+    "realtimeTranscriptionId": "s-2a913f94-93e372e2-60da-4c89-beb0-0d3a219b287c",
+    "transcriptionName": "live_audience",
+    "tracks": [
+      {
+        "name": "inbound",
+      },
+      {
+        "name": "outbound",
+      }
+    ]
+  },
+  "customParams": {
+    "foo": "bar",
+    "foos": "bars"
+  }
+}
+```
+
+### A `transcription` Websocket Message
+```json
+{
+  "eventType": "transcription",
+  "track": "inbound",
+  "startTime": "2023-03-31T20:05.101Z",
+  "endTime": "2023-03-31T20:07.493Z",
```

Comment:
AWS gives us the time in seconds since we started sending audio (their docs says millis but I doubt it as per my testing). Wonder if that makes more sense than clock time?